dotcms-integration/pom.xml

```diff
@@ -79,6 +79,7 @@
         <maven.failsafe.debugArgs/>
         <coreit.test.skip>true</coreit.test.skip>
         <docker.filter>database,opensearch</docker.filter>
+        <jackson.version>2.17.0</jackson.version>
```

Comment:
This is going to override any version of jackson we are using in the application bom and could cause us issues if there are conflicts.   This is only for test scope some not as bad but would be better if we can make sure the jackson version is consistent and upgrade our default one.  If that can be done then we shoud not have the jackson.version in this file or specify the version tag.  Or base version is currently 2.16.1,  so worth seeing if we can upgrade to 2.17.0 if that is needed here.

Reply:
@spbolton I am aware of dependencies in bom project but it didn't work because of WireMock relies on that one. I didn't foreseen any issues since this is only for integration tests at `test` scope.